tensorflow/lite/micro/docs/porting_reference_ops.md

```diff
@@ -0,0 +1,267 @@
+for the specific task of porting reference ops from Lite to Micro.
+
+<!--
+Semi-automated TOC generation with instructions from
+https://github.com/ekalinin/github-markdown-toc#auto-insert-and-update-toc
+-->
+<!--ts-->
+* [Porting Reference Ops from Lite to Micro](#porting-reference-ops-from-lite-to-micro)
+   * [1. Look for a port already in progress](#1-look-for-a-port-already-in-progress)
+   * [2. Open a GitHub issue to track the port](#2-open-a-github-issue-to-track-the-port)
+   * [3. Extract Lite's code for parsing op parameters to a function (PR1)](#3-extract-lites-code-for-parsing-op-parameters-to-a-function-pr1)
+   * [4. Extract the reference for the op to a standalone header (PR2)](#4-extract-the-reference-for-the-op-to-a-standalone-header-pr2)
+   * [5. Port the op from Lite to Micro (PR3)](#5-port-the-op-from-lite-to-micro-pr3)
+* [General Guidelines](#general-guidelines)
+   * [Check each commit for formatting, lint, and unit-test passage](#check-each-commit-for-formatting-lint-and-unit-test-passage)
+   * [Add yourself as an asignee to PRs](#add-yourself-as-an-asignee-to-prs)
+   * [Maintain a 1:1 correspondence between Micro and Lite versions of unit tests](#maintain-a-11-correspondence-between-micro-and-lite-versions-of-unit-tests)
+   * [Sometimes CI checks on PRs are flakey and fail](#sometimes-ci-checks-on-prs-are-flakey-and-fail)
+* [Notes](#notes)
+
+<!-- Added by: rkuester, at: Wed 05 May 2021 07:44:11 PM CDT -->
+
+<!--te-->
+
+## 1. Look for a port already in progress
+
+Begin by searching the TensorFlow GitHub repository for issues containing the
+name of the op under consideration to ensure someone isn't already working on a
+port.
+
+## 2. Open a GitHub issue to track the port
+
+Open a GitHub issue to announce your intent to port the op, and to begin a
+record of your work. Document the entire process of porting the op in this
+issue. Link constituent PRs to this issue. See the article [Providing
+Context][] for background on documenting your work via bug reports.
+
+A good example is [issue #45306: micro: port op FILL from lite](https://github.com/tensorflow/tensorflow/issues/45306).
+
+## 3. Extract Lite's code for parsing op parameters to a function (PR1)
+
+Now we begin changing, testing, and submitting code. This step will result in
+the first pull request, PR1.
+
+1.  Extract the code for parsing op parameters out of the switch statement in
+    [`ParseOpDataTfLite()`][] in `lite/core/api/flatbuffer_conversions.cc` into
+    a standalone function, and call that function from the switch statement.
+    This standalone function is now available to be called by the Micro op
+    resolver, which also needs to parse the op parameters, in a future change.
+    A simple example is [PR #45307][], and a more complicated example is [PR
+    #46021][].
+
+1.  Use `clang-format` to make sure the code is properly formatted.
+
+        clang-format --style=google -i $(git ls-files -m | grep -E '\.cc|\.h')
```

Comment:
Small suggestion - for command lines, you might consider using code fences (https://developers.google.com/style/code-syntax?hl=en)/code blocks (https://www.tensorflow.org/community/contribute/docs_style#write_about_code), specifying the "language" (e.g. `shell`, `python`, etc):

e.g.

```
    ```shell
    python3 awesome.py --style=awesome
    ```
````

Reply:
I've done as you suggested in 5db4106, and further changed some formatting per the style guides you cited. Thanks for the review.

Reply:
No worries, glad I could help